app/models/force/object_base.rb

```diff
@@ -73,16 +73,35 @@ def currency_to_float(field_name, salesforce_fields)
       salesforce_fields[field_name] = salesforce_fields[field_name].gsub(/[$,]/, '').to_f
     end
 
+    # TODO: Automate this conversion for date objects.
     def self.date_to_json(api_date)
       return nil if api_date.blank?
       date = api_date.split('-')
       { year: date[0], month: date[1], day: date[2] }
     end
 
+    # TODO: Automate this conversion for date objects.
     def self.date_to_salesforce(domain_date)
       return nil unless !domain_date.blank? && %i[year month day].all? {|s| domain_date.key? s}
       lease_date = Date.new(domain_date[:year].to_i, domain_date[:month].to_i, domain_date[:day].to_i)
       lease_date.strftime('%F')
     end
+
+    def self.get_domain_keys(obj = self)
```

Comment:
I created this so that we could automate which fields we permit in the short form controller